astropy/coordinates/representation.py

```diff
@@ -409,6 +547,85 @@ def transform(self, matrix):
         newxyz = u.Quantity(newxyz, oldxyz.unit, copy=False)
         return self.__class__(*newxyz, copy=False)
 
+    def _combine_operation(self, op, other, reverse=False):
+        try:
+            other_c = other.to_cartesian()
+        except Exception:
+            return NotImplemented
+
+        first, second = ((self, other_c) if not reverse else
+                         (other_c, self))
+        return self.__class__(*(op(getattr(first, component),
+                                   getattr(second, component))
+                                for component in first.components))
+
+    def mean(self, *args, **kwargs):
+        """Return the vector mean along a given axis.
```

Comment:
Again a good point.  I removed "along the given axis" as I agree it is only confusing, and expanded text as follows:
```
        """Vector mean.

        Averaging is done by converting the representation to cartesian, and
        taking the mean of the x, y, and z components. The result is converted
        back to the same representation as the input.

        Refer to `~numpy.mean` for full documentation of the arguments, noting
        that ``axis`` is the entry in the ``shape`` of the representation, and
        that the ``out`` argument cannot be used.
```